content/clients/index.md

```diff
@@ -63,18 +63,24 @@ At the writing time - more recent, compared to some other gtk clients.
 
 [Skympc](https://github.com/soramimi/SkyMPC) - Yet anthoer QT(5) Client
 
+[Theremin](https://github.com/kampfgnu/Theremin) A MDP client in Objective-C with binaries for MacOS (at least up to High Sierra)
+
 [Xfce4-mpc-plugin](https://goodies.xfce.org/projects/panel-plugins/xfce4-mpc-plugin) - Plugin for Xfce4 Panel
 
 [Xfmpc](https://goodies.xfce.org/projects/applications/xfmpc) - A graphical GTK+ MPD client focusing on low footprint
 
 [YMPD](https://github.com/notandy/ympd) - Standalone MPD Web GUI written in C, utilizing Websockets and Bootstrap/JS
 
-## Android
+## Android / iOS
+
+[M.A.L.P.](https://gitlab.com/gateship-one/malp) - A MPD client for Android
 
-[M.A.L.P.](https://gitlab.com/gateship-one/malp) - A mpd client for Android
+[MaximumMPD](https://github.com/rbackhouse/MaximumMPD) - A MDP client for iOS
 
 [MPDroid](https://github.com/abarisain/dmix) - A modern MPD Client for Android
 
+[Soundirok](https://www.kvibes.de/soundirok/) - A commercial mpd client for Android and iOS with a targeted at Volumio and Runeaudio.
```

Comment:
Non-free software will not be recommended on the MPD website.

Reply:
Ah, I guess in that case it might be helpful for both readers and contributors to point that out somewhere. Especially since there IS a list of commercial products using MDP on your commercial page.

Reply:
I created an issue about that.

Reply:
The list of commercial products is about hardware; you buy the hardware, and it comes with free software. But there are problems with the free software: it is usually horribly outdated and contains badly written (or illegal) patches. It should be obvious that those products cannot be recommended.

Reply:
I don't follow. But maybe I was unable to express what I mean. Please bear with me while I try to explain: I found the MDP site while searching for an MDP client my wife might use to control our rune audio player. There was a list of clients, and a message on the bottom that asked for help adding more players. I had just found and tested a few that were missing on the page, so I considered adding them. I briefly scanned the page for clues what is and is not appropriate, and found the commercials page. Since I found no other clue, I added a commercial client. You said that is not good. I suggested to make that clear before the next guy comes along and wastes their time, and yours, again.

Reply:
Your bug report was good, you have a point, and I added a clarification.
My last reply here was just trying to explain why the "commercial" page lists commercial products, while at the same time I consider non-free clients not acceptable for the "clients" page.